webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -10,9 +10,10 @@
     graphic logo is a trademark of OpenMRS Inc.
 
 -->
-<web-app xmlns="http://java.sun.com/xml/ns/javaee"
-		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-Instance"
-		 xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
+
```

Comment:
Can you revert the changes to this file?

Reply:
Did you see the above comment?

Reply:
I did revert but each time I rebase, the file is changed. It seems these changes have been made to master

Reply:
No merge has been done on master. If i file changes accidentally, you do not have to commit it. Can you ask your team mates for help? We just need a pull request which does not have changes to this file.